helm/prometheus-operator-app/Chart.yaml

```diff
@@ -6,6 +6,7 @@ icon: https://s.giantswarm.io/app-icons/prometheus-operator/2/light.svg
 version: [[ .Version ]]
 sources:
   - "https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack"
+  - "https://github.com/giantswarm/grafana-app/tree/master/helm/grafana"
 home: https://github.com/giantswarm/prometheus-operator-app
```

Comment:
Know that this will require a really explicative changelog and major version bump :(

Reply:
Should I still go on with this then ?

Reply:
Outdated due to recent changes